organizers/2024-03-13.html

```diff
@@ -0,0 +1,148 @@
+</section>
+
+<section>
+    <h1>👋 Welcome All!</h1>
+</section>
+
+<section>
+    <h1>Code of conduct</h1>
+    <h2>😃 Be Awesome to Each Other</h2>
+    <p>
+        The mission of The London Django Meetup Group is to provide a fun,
+        welcoming and professional environment for a diverse group of people.
+    </p>
+    <p>
+        Code of conduct:
+        <a href="https://www.djangolondon.com/code-of-conduct/">djangolondon.com/code-of-conduct</a>
+    </p>
+</section>
+
+<section>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Kraken Tech (part of the Octopus Energy Group)</strong></li>
+        <li><strong>Jetbrains</strong></li>
+    </ul>
+</section>
+
+<section>
+    <img data-src="/img/2022_KrakenV2_stacked_White.png"
+         style="width: 60%; border-radius: 10px"
+         alt="Kraken Tech"
+         class="plain">
+    <p>
+        <em>
+            Kraken is a revolutionary energy tech platform built with Django. Serving 25 million households worldwide.
+        </em>
+    </p>
+</section>
+
+<section>
+    <h1>📋 Today's Schedule</h1>
+    <h2>🐍 News</h2>
+    <h2>👩🏻‍💻 Talks</h2>
+    <h2>🎁 Prize Draw</h2>
+    <h2>💬 Open Discussion</h2>
+</section>
+
+<section>
+    <h1>Django News</h1>
+    <p>Weekly newsletter of blog posts, developments, talks, packages, etc.</p>
+    <p>django-news.com</p>
+</section>
+
+<section>
+    <h2>🔒 Django security releases</h2>
+    <p>5.0.3 and related backports fix a DoS vulnerability.</p>
+    <p>Upgrade monthly!</p>
+</section>
+
+<section>
+    <h2>🇪🇸 Djangocon Europe</h2>
+    <p>June 5th to 9th</p>
+    <p>Vigo, Spain</p>
+    <p>🐣 Earlybird tickets still on sale.</p>
+</section>
+
+<section>
+    <h2>🇺🇸 Djangocon US CfP</h2>
+    <p>Deadline: April 24</p>
+    <p>Submit your talk or workshop proposal now!</p>
+</section>
+
+<section>
+    <h2>🇨🇿  EuroPython 2024</h2>
+    <p>July 8-14, 2024</p>
+    <p>Prague, Czech Republic</p>
+    <p>Volunteer opportunities open, CfP until <strong>tomorrow morning</strong>.</p>
+</section>
+
+<section>
+    <h2>🇮🇹 PyCon Italia 2024 - Seeking Sponsors!</h2>
+    <p>Dates: May 22-25, 2024</p>
+    <p>Florence, Italy</p>
+    <p>Join over 800 attendees from 45 countries for 100+ talks and workshops.</p>
+</section>
+
+<section>
+    <h1>🎤 Talks</h1>
+    <strong>WASM-powered Django apps with PyScript</strong><br>
+       <em>Valerio Maggio</em>
+    <strong>Navigating your job search in 2024</strong><br>
+       <em>Jon Gould</em>
+</section>
+
+<section>
+    <h1>🎁 Prize Draw</h1>
+    <p>
+        You have a chance to win one of two 100% discount codes for a JetBrains
+        product (AppCode, PyCharm, WebStorm. etc)
+    </p>
+    <p>
+        Put your name in the hat and we will draw the winners and send the licences via email.
+    </p>
+</section>
+
+<section>
+    <h1>Want to give a talk? 🤔</h1>
+    <p>Please use the form linked on:</p>
+    <ul>
```

Comment:
![QR Code Generator](https://github.com/djangolondon/djangolondon.github.io/assets/28976199/fd688437-f10d-4e42-9232-7db5ec186536)

What do you think about adding a qr-code so people can easy scan and don't forget (hopefully) to submit their talks? 
(Can be done in the next month slides as well!)

Reply:
Good idea, sorry I missed it. Let’s try again next month.

Reply:
Added in #175